plugins/postcss.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (ctx) => ({
+  map: ctx.options.map,
+  parser: ctx.options.parser,
+  plugins: {
+    'postcss-import': { root: ctx.file.dirname },
+    'postcss-nested': { },
+    'postcss-cssnext': {
```

Comment:
We can remove this for now. We don't have that many colors, so we don't need variables atm.